polaris.shopify.com/scripts/get-styles.mjs

```diff
@@ -0,0 +1,10 @@
+import fs from 'fs';
+
+const builtStylesPath = '../polaris-react/build/esm/styles.css';
```

Comment:
I'm assuming Polaris is always built when `yarn dev` runs, otherwise other things would fail as well, right?

Reply:
Only if it happens at the root of the monorepo. This could be a little bit confusing if someone is running just the website.